contracts/token-distribution/SaleRounds.sol

```diff
@@ -12,18 +12,24 @@ contract SaleRounds is TokenDistribution, GameOwner, ERC20 {
     using SafeMath for uint;
     using Math for uint;
 
-    mapping(RoundType => bool) private activeRound;
+    struct ClaimInfo {
+        uint cliff;
+        uint vesting;
+        uint balance;
+        uint claimedBalance;
```

Comment:
This one can probably live in local function memory? Depends on style preference maybe, feels dirty to me.